google/cloud/bigquery/job/query.py

```diff
@@ -1620,6 +1625,10 @@ def to_dataframe(
         create_bqstorage_client: bool = True,
         max_results: Optional[int] = None,
         geography_as_object: bool = False,
+        bool_dtype: Union[Any, None] = pandas.BooleanDtype(),
```

Comment:
Likewise, we can't initialize pandas objects at import time since pandas is an optional dependency.

Reply:
Addressed. Thanks!